staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1767,7 +1767,7 @@ type CSIPersistentVolumeSource struct {
 	// +optional
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,3,opt,name=readOnly"`
```

Comment:
looks like this needs a godoc starting with `readOnly ...`

Reply:
Sure, done.